RPIPs/RPIP-10.md

```diff
@@ -29,72 +29,71 @@ changes to how the funds should be directed.
   - Incentives (e.g., LP bonuses) - 50%
   - Grants and Bounties - 30%
   - Reserve Treasury - 20%
-- Where a matching MC exists (eg, the "Incentives Management Committee" for the "Incentives"
-  category), incoming funds SHALL be disbursed to the matching MC as quickly as practicable.
+- Where a matching Management Committee exists (eg, the "Incentives Management Committee" for the "Incentives"
+  category), incoming funds SHALL be disbursed to the matching Management Committee as quickly as practical.
```

Comment:
I believe the word should remain as practicable. 
<https://grammarist.com/usage/practical-practicable/#:~:text=Think%20of%20practical%20as%20a,applies%20to%20plans%20or%20actions.>

I think we want to use the "feasible" version.

Reply:
Discussed a little bit internally. I quite like practicable, but it's now been too many smart people needing to fish out thesauruses for too little semantic gain -- going with "practical".